### PR TITLE
Master prepare new env 2 seb

### PR DESCRIPTION
**Pull request by seb-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mail/static/src/models/chat_window/chat_window.js [hunk 1]
```diff
@@ -20,22 +20,24 @@ function factory(dependencies) {
...
+                this.messaging.messagingMenu.update({ isOpen: true });
+            }
+            // Flux specific: 'closed' fold state should only be saved on the
+            // server when manually closing the chat window. Delete at destroy
+            // or sync from server value for example should not save the value.
+            if (this.thread && notifyServer) {
+                this.thread.notifyFoldStateToServer('closed');
+            }
+            if (this.exists()) {
+                // notifyFoldStateToServer() might have deleted it already
```
Comment (alexkuhn) [inline addons/mail/static/src/models/chat_window/chat_window.js:39]: "might" => "might not"?

Comment (alexkuhn) [reply addons/mail/static/src/models/chat_window/chat_window.js:39]: Oops, didn't meant to post comment yet, but I think I prefer no comment here.

Comment (seb-odoo) [reply addons/mail/static/src/models/chat_window/chat_window.js]: done

**Review by alexkuhn — APPROVED** [review]
@robodoo r+

**Comment by seb-odoo** [comment]
@robodoo rebase-ff

**Pull request closed without merging**